appveyor.yml

```diff
@@ -46,9 +46,9 @@ test_script:
 cache:
   - '%APPDATA%\npm-cache'
   - '%LOCALAPPDATA%\pip\Cache'
-  # - C:\ProgramData\chocolatey\bin
-  # - C:\ProgramData\chocolatey\lib
-  # - C:\tools\php81
+  - C:\ProgramData\chocolatey\bin -> appveyor.yml
```

Comment:
Just adding them here will not work. It will only used the cached versions on the second run, so the first build will succeed, but the consequent builds will fail.